FEM/rf_pcs.cpp

```diff
@@ -3200,6 +3200,10 @@ void CRFProcess::VariableStaticProblem()
 	pcs_secondary_function_unit[pcs_number_of_secondary_nvals] = "-";
 	pcs_secondary_function_timelevel[pcs_number_of_secondary_nvals] = 1;
 	pcs_number_of_secondary_nvals++;
+    pcs_secondary_function_name[pcs_number_of_secondary_nvals] = "DILATANCY";
```

Comment:
indentation
It is better to introduced  a if-condition for this variable in order to save memory for the problems that do not use this variable.